docs/scalafix-migrations.md

```diff
@@ -31,21 +31,21 @@ which resolves those rules and edits the source code accordingly.
 See the [Scalafix Developer Guide][scalafix-dev-guide] for more information
 about writing rewrite rules or have a look at the existing
 [migration rules][migrations] for inspiration. Rules in Scala Steward must be
-accessible via the [github:][using-github], the [http:][using-http] or the [dependency:][using-dependency] schemes.
+accessible via the [github:][using-github], the [http:][using-http] or the
+[dependency:][using-dependency] schemes.
 
 ## Adding migration rules to Scala Steward
 
 After you have written a new migration rule for a new version of your project,
 Scala Steward needs to be made aware of it. Creating a pull request that adds
 the new rule to the list of [migrations][migrations] is enough for that. Once
 that pull request is merged, Scala Steward will start using this migration.
-When running Scala Steward you can specify a file containing extra migrations
-that might not be present in the [default list][migrations].
-You can also specify if you want the default list to be disabled.
 
-The file is in [HOCON][HOCON] and should look like this:
+When running Scala Steward you can also specify files via the
+`--scalafix-migrations` command-line option that contain additional
```

Comment:
```suggestion
`--scalafix-migrations` command-line option that contains additional
```

Reply:
The subject this belongs to is "files or URLs".